22_markers/FindMarkers.Rmd

```diff
@@ -49,5 +49,34 @@ find_markers("Spleen", "droplet", "cell_ontology_class")
 
 find_markers("Thymus", "facs", "cell_ontology_class")
 find_markers("Thymus", "droplet", "cell_ontology_class")
+
+find_markers("Brain_Non-Myeloid", "facs", "cell_ontology_class")
 ```
 
+```{r}
+library(openxlsx)
```

Comment:
I'm not up to speed on the recent changes to this project overall so this might be a silly questions: what is the reason to list cell type markers by using an excel workflow?

If we do want to allow an excel workbook would we also want to support csv or another format? If not then ignore the rest, otherwise I would probably split this function to one that creates a dataframe with the needed values and returns that. Then create a separate function that would create an excel workbook given that dataframe. Then if someone wanted to write csvs or just explore the dataframe the function can be used for those purposes. Again this might not make any sense given the use case.